hooks/tk-maya_actions.py

```diff
@@ -243,7 +243,7 @@ def _create_udim_texture_node(self, path, sg_publish_data):
             mel.eval("generateUvTilePreview %s" % file_node)
         return file_node
 
-    def _create_image_plane(self, path, sg_publish_data):
+    def _create_image_plane(self, path_utf8, sg_publish_data):
```

Comment:
same here - if image planes require different unicode input than udim texture nodes and other maya things, we should add some really clear comments in the code explaing this. If not, we should lift up the conversion a level (line 157), so we only do it once for all the maya logic